Documentation/network/bgp-control-plane.rst

```diff
@@ -364,6 +364,9 @@ with a dummy key and value can be used like:
          matchExpressions:
             - {key: somekey, operator: NotIn, values: ['never-used-value']}
 
+A service must have `loadBalancerClass <https://kubernetes.io/docs/concepts/services-networking/service/#load-balancer-class>`__
+unspecified or set to "io.cilium/bgp-control-plane" to be announced by a policy.
```

Comment:
Nit: I'd rather have pre-formatted text to mark that this is a specific string here,

```suggestion
unspecified or set to ``io.cilium/bgp-control-plane`` to be announced by a policy.
```

Reply:
Commit 0c5f501 resolves this comment.